apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -702,9 +720,9 @@ async function legacyTraverseQuestionAndExecuteFunctions(phase, codeCaller, data
 
         const elementFile = getElementController(elementName, context);
         // Populate the extensions used by this element
-        data.extensions = [];
+        /** @type {any} */ (data).extensions = [];
```

Comment:
This feels like a workaround, is there a better way to do this? I always thought the `delete extensions` was strange code.

Reply:
We could assign `undefined` instead if you prefer?

FWIW, we do this same little thing in the Python processor:

https://github.com/PrairieLearn/PrairieLearn/blob/4099a2df4f77c1da59e6580ab55c0312de6d0e65/apps/prairielearn/python/prairielearn/internal/question_phases.py#L214-L248

Reply:
I would prefer to do a spread operation, like

```
{
...data,
extensions: foo
}
```

This is a nit btw, not blocking.

Reply:
That's a reasonable suggestion! I'll have better types around element extensions in a few minutes once #11828 lands; I can rework this then.

Reply:
Sounds good. Feel free to resolve this comment.

Reply:
Resolved in 1fe0247.